keras_cv/models/resnet_v1_test.py

```diff
@@ -14,7 +14,9 @@
 
 import tensorflow as tf
 from absl.testing import parameterized
-from packaging import version
+
+from packaging.version import Version
+
```

Comment:
There are 2 empty lines around `from packaging.version import Version`.

`./shell/format.sh` can fix this automatically